lib/msal-node/test/client/ClientAssertion.spec.ts

```diff
@@ -3,6 +3,7 @@ import { TEST_CONSTANTS } from "../utils/TestConstants";
 import { CryptoProvider } from "../../src/crypto/CryptoProvider";
```

Comment:
Can you please add tests for AcquireTokenSilent and AcquireTokenByAuthorization code for this? These are both part of confidential client.

Reply:
I think you're looking at the wrong file. The test for AcquireTokenByAuthorization already exists in ConfidentialClientApplication.spec.ts. However, a AcquireTokenSilent is missing from that file. I will add one.